cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -75,6 +75,45 @@ Replaced by:
 ----
 DROP CONSTRAINT name [IF EXISTS]
 ----
+
+
+a|
+label:syntax[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
+FOR ...
+REQUIRE ... IS NOT NULL
+OPTIONS "{" "}"
+----
+a|
+Existence constraints now allow an `OPTIONS` map, at this point there are no available values for the map.
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
+FOR ... ON ...
+OPTIONS "{" option: value[, ...] "}"
+----
+a|
+Token lookup indexes now allow an `OPTIONS` map to specify the index provider.
+
+
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE TEXT INDEX ...
```

Comment:
update the btree deprecation part to show what part is replaced by text indexes once the btree deprecation pr is in

```
a|
label:functionality[]
label:deprecated[]
[source, cypher, role="noheader"]
----
CREATE BTREE INDEX ...
OPTIONS "{" indexProvider: 'lucene+native-3.0' "}"
----
a|
B-tree indexes are deprecated and will need migration before upgrading to 5.0.

B-tree indexes with the `lucene+native-3.0` index provider is replaced by:
[source, cypher, role="noheader"]
----
CREATE TEXT INDEX ...
----
```